data/container-workload.service

```diff
@@ -3,6 +3,7 @@ Description=Container Workload
 After=opt-rootfs-resolv.service
 After=opt-rootfs-proc.mount
 After=opt-rootfs-sys.mount
+After=hyperstart.service
```

Comment:
this needs to be looked at in more details, the STARTPOD/NEWCONTAINER hyperstart commands seem to do some setup in this area, something we'll need to do at "create" time in the runtime. Investigation still needed.